tools/gatk_intervallisttool.cwl

```diff
@@ -54,15 +59,15 @@ arguments:
       }
 
 inputs:
-  interval_list: File
+  interval_list: {type: 'File?'}
   bands: int
   scatter_ct: int
   reference_dict: {type: ['null', File], doc: "Provide only if input is bed file instead of gatk style .interval_list", sbg:suggestedValue: {name:  "Provide only if input is bed file instead of gatk style .interval_list"}}
   exome_flag: {type: ['null', string], doc: "If 'Y', will set bands to 0 to prevent breaking up of intervals", default: "N"}
   break_by_chr: {type: ['null', string], doc: "If Y, break up files by chr.  If creating smaler intervals, recommend scatter_ct=1", default: "N"}
 outputs:
   output:
-    type: 'File[]'
+    type: 'File[]?'
```

Comment:
Hmmm, downstream tools won't work with no input, unless the intent is to make it more flexible in other pipes where splitting and/or lists are optional...

Reply:
This could be a problem for mutect2. All other instances of an empty output would be caught elsewhere before they have a chance to affect data.

Reply:
I've added a new check for this